tests/conftest.py

```diff
@@ -221,22 +225,34 @@ def enable_ssh_timout(dut):
     time.sleep(5)
 
 
+@pytest.fixture(name="duthosts", scope="session")
+def fixture_duthosts(ansible_adhoc, testbed):
```

Comment:
any particular reason, you selected this naming convention? I personally prefer the style that is used throughout our repo since the @pytest.fixture itself implies that the following method is a fixture

@pytest.fixture(scope="")
def duthosts(args):

Reply:
Because if a fixture(fixture `duthosts`) is used in the same module(by fixture `duthost`), it will be shadowed by its function(function `duthosts`).